eclcli/bare/v2/server.py

```diff
@@ -548,13 +548,20 @@ def get_parser(self, prog_name):
             metavar="<server>",
             help="Name or ID of server",
         )
+        parser.add_argument(
+            "--type",
+            metavar="<type>",
+            default="SOFT",
```

Comment:
If using SOFT as default value, this will not work in regions that don't have new baremetal API.